packages/nuxt/src/runtime/plugins/sentry.server.ts

```diff
@@ -0,0 +1,23 @@
+import { captureException } from '@sentry/node';
+import { H3Error } from 'h3';
+import { defineNitroPlugin } from 'nitropack/runtime';
+
+export default defineNitroPlugin(nitroApp => {
+  nitroApp.hooks.hook('error', (error, context) => {
+    // Do not handle 404 and 422
+    if (error instanceof H3Error) {
+      if (error.statusCode === 404 || error.statusCode === 422) {
+        return;
+      }
+    }
+
+    if (context) {
+      captureException(error, {
+        captureContext: { contexts: { nuxt: context } },
```

Comment:
m: I would be more selective in what we record here instead of dumping the entire context object in here. It could become a problem if context suddenly starts containing PII, becomes very large, or circular in a future nitro update.

Reply:
yes ideally we:
1. We parse the incoming nuxt context and extract the fields we care about
2. We put that in a nuxt context attached to the sentry event
3. Document the nuxt context in the develop docs